lib/elixir/lib/code.ex

```diff
@@ -1095,7 +1095,7 @@ defmodule Code do
       Defaults to 1.
 
     * `:columns` - when `true`, attach a `:column` key to the quoted
-      metadata. Defaults to `false`.
+      metadata. Defaults to `true`.
```

Comment:
Should we document that the default was different up to Elixir 1.15?

Reply:
We could. We could also put it in breaking changes - some libraries pattern match on AST metadata and break when compiling with `:columns` - at least 2 such cases were reported to elixir-ls

Reply:
We didn't change the value here. We changed it on parser_options which I already fixed in another commit. :) This one still remains the same!

Reply:
Oh, I didn't notice the defaults are now different in those 2 cases. @josevalim what about my comment on documenting the breaking change?